docs/introduction-to-smart-contracts.rst

```diff
@@ -33,9 +33,9 @@ Storage
 The first line simply tells that the source code is written for
 Solidity version 0.4.0 or anything newer that does not break functionality
 (up to, but not including, version 0.5.0). This is to ensure that the
-contract does not suddenly behave differently with a new compiler version.
+contract does not suddenly behave differently with a new compiler version. For more information about pragma, see [here](https://en.wikipedia.org/wiki/Pragma_once).
```

Comment:
My intent is to make it easier for someone who has no prior experience with programming to learn Solidity as their first programming language. While the suggestion may not be best, it would be helpful to explain more about pragma for those who may want to know.